### PR TITLE
Add class_name/recipe fields to Operator model

### PR DESCRIPTION
**Pull request by aef-**
Screen Shot 2020-01-23 at 14 41 17

### CODE REVIEW

File: runner/tasks.py [hunk 1]
```diff
@@ -3,18 +3,39 @@
...
+    jobs = operator.get_jobs()
+    for job in jobs:
+        if job[0].is_valid():
+            logger.info("Creating Run object")
+            run = job[0].save()
+            logger.info("Run object created with id: %s" % str(run.id))
+            create_run_task.delay(str(run.id), job[1], None)
+        else:
+            logger.error("Job invalid: %s" % str(job[0].errors))
+
```
Comment (aef-) [inline runner/tasks.py]: This is all copied from `operator_job`

File: beagle_etl/views.py [hunk 2]
```diff
@@ -56,17 +56,11 @@ def post(self, request):
...
-                job.save()
-            else:
-                job = Job(run='beagle_etl.jobs.lims_etl_jobs.fetch_samples', args={'request_id': request_id},
-                          status=JobStatus.CREATED, max_retry=1, children=[])
-                job.save()
+            job = Job(run='beagle_etl.jobs.lims_etl_jobs.fetch_samples', args={'request_id': request_id},
+                      status=JobStatus.CREATED, max_retry=1, children=[],
+                      callback='beagle_etl.jobs.lims_etl_jobs.request_callback',
+                      callback_args={'request_id': request_id})
+            job.save()
```
Comment (aef-) [inline beagle_etl/views.py]: `active` is checked in `request_callback` per `Operator`

File: beagle_etl/models.py [hunk 3]
```diff
@@ -33,4 +33,7 @@ class Job(BaseModel):
 
 
 class Operator(models.Model):
+    slug = models.CharField(max_length=100, default=False)
+    class_name = models.CharField(max_length=100)
     active = models.BooleanField(default=False)
+    recipes = ArrayField(models.CharField(max_length=50, default=False))
```
Comment (aef-) [inline beagle_etl/models.py]: `slug` can be used like hardcoded `pipeline_name`

File: runner/migrations/0013_merge_20200117_2050.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.8 on 2020-01-17 20:50
```
Comment (sivkovic) [inline runner/migrations/0013_merge_20200117_2050.py]: We have this on develop branch now https://github.com/mskcc/beagle/blob/develop/runner/migrations/0013_merge_20200117_1842.py. I am not sure can duplicate cause us problems now?

**Comment by stevekm** [comment]
started some tests for Operator here; https://github.com/mskcc/beagle/pull/114

**Pull request merged**